docs/install/installEverest.md

```diff
@@ -29,7 +29,134 @@ Percona Everest assists with installing all the necessary operators and required
+    ```sh
+    kubectl create namespace percona-everest
+    ```
+2. Deploy Everest to Kubernetes:
+    ```sh
+    kubectl apply -f https://raw.githubusercontent.com/percona/percona-everest-backend/v0.4.0/deploy/quickstart-k8s.yaml -n percona-everest
+    ```
+3. (Optional) Verify that the services started correctly:
+    
+    ```sh
+    kubectl get pods -n percona-everest
+    ```        
+    ??? example "Expected output"
+        ```
+        NAME                                                   READY   STATUS    RESTARTS       AGE
+        percona-everest-0                                      2/2     Running   2 (10s ago)    10s
+        ```
+
+4. Make a note of the external IP address for the Everest service (for instance, 127.0.0.1 in this example):
+   
+    ```sh 
+    kubectl get svc/everest -n percona-everest
+    ```
+    
+    ??? example "Expected output"
+        ```
+        NAME      TYPE           CLUSTER-IP      EXTERNAL-IP     PORT(S)          AGE
+        everest   LoadBalancer   10.43.172.194   127.0.0.1       8080:31611/TCP   10s
+        ```
+
+5. Rename the downloaded file using the following command and replacing the placeholder `everestctl-darwin-amd64` to match the file downloaded in the previous step:
+    
+    ```sh
+    mv everestctl-darwin-amd64 everestctl
+    ```
+
+6. Modify the file permissions:
+
+    ```sh
+    chmod +x everestctl
+    ```
+    
+7. From the installation wizard, provision and register the Kubernetes cluster in Everest using the following commands. Everest will search for the kubeconfig file in the `~/.kube/config` path. If your file is located elsewhere, set the `KUBECONFIG` environment variable before running the command:
+  
+    ```sh
+    export KUBECONFIG=~/.kube/config
+    ```
+
+  
+    ```sh
+    ./everestctl install operators
+    ```
+    
+    This will install all needed components in a namespace called `percona-everest`.
+
+    ??? example "Example"
+            
+            ? Everest URL http://127.0.0.1:8080
+            ? Choose your Kubernetes Cluster name k3d-dbaas-tilt
+            ? Do you want to enable monitoring? Yes
+            ? PMM URL Endpoint http://127.0.0.1
+            ? Username admin
+            ? Password *****
+            ? Name for the new monitoring instance my-pmm
+            ? Do you want to enable backups? No
+            ? What operators do you want to install? MySQL, MongoDB, PostgreSQL
+            
+
+    Alternatively, you can provision and register the Kubernetes cluster by running the installation in headless mode:
+        
+    ```
+    ./everestctl install operators --backup.enable=false --everest.endpoint=http://127.0.0.1:8080 --operator.mongodb=true --operator.postgresql=true --operator.xtradb-cluster=true --monitoring.enable=true --monitoring.type=pmm --monitoring.new-instance-name=my-pmm --monitoring.pmm.endpoint=http://127.0.0.1 --monitoring.pmm.username=admin --monitoring.pmm.password=admin  --skip-wizard
+    ```
```

Comment:
I don't know exactly what's the best place to say this but we should inform the users that the `Everest URL` (if using the using) or `--everest.endpoint=` (if running headless) should use the external IP value we got from step 4, 127.0.0.1 is just an example.